corehq/apps/hqwebapp/crispy.py

```diff
@@ -352,6 +353,10 @@ class RadioSelect(Field):
     template = "hqwebapp/crispy/radioselect.html"
 
 
+class DatetimeLocalWidget(DateTimeInput):
+    input_type = "datetime-local"
```

Comment:
added a new widget to modify input_type needed to use HQ's plugged in use of momentJS as also used in the admin/superuser page where we add alerts for all of HQ.